/*
 * ConfigInc.h
 *
 *  Created on: 2016-8-22
 *      Author: Ralf
 */

#ifndef CONFIGINC_H_
#define CONFIGINC_H_

#include "CfgCommon.pb.h"
#include "CfgMap.pb.h"
#include "CfgDungeon.pb.h"
#include "CfgMonster.pb.h"
#include "CfgSkill.pb.h"
#include "CfgHang.pb.h"
#include "CfgLevel.pb.h"
#include "CfgMission.pb.h"
#include "CfgAdvance.pb.h"
#include "CfgItem.pb.h"
#include "CfgBag.pb.h"
#include "CfgAttrDef.pb.h"
#include "CfgForge.pb.h"
#include "CfgCard.pb.h"
#include "CfgTreasure.pb.h"
#include "CfgSuit.pb.h"
#include "CfgPurify.pb.h"
#include "CfgRing.pb.h"
#include "CfgReinCarn.pb.h"
#include "CfgMagic.pb.h"
#include "CfgCopy.pb.h"
#include "CfgBoss.pb.h"
#include "CfgActivity.pb.h"
#include "CfgTitle.pb.h"
#include "MsgLocal.pb.h"
#include "CfgZhuLing.pb.h"
#include "CfgZhanLing.pb.h"
#include "CfgOnlineReward.pb.h"
#include "CfgSignRewards.pb.h"
#include "CfgReinCarnSheng.pb.h"
#include "CfgTrump.pb.h"
#include "CfgFashion.pb.h"
#include "CfgFashionSuit.pb.h"
#include "CfgPartnerIsland.pb.h"
#include "CfgShop.pb.h"
#include "CfgMoney.pb.h"
#include "CfgConst.pb.h"

#endif /* CONFIGINC_H_ */
